### ISSUE TITLE
ISSUE-3: Style button for adding new entries

### ISSUE THREAD

**Issue by davefcodes** [davefcodes/Mind-Expanders#3]

**Comment by tushar1-000** [comment]
Can you elaborate more?

**Comment by davefcodes** [comment]
Yes I planned on styling the button on the entries page. I have attached a screenshot. 
![Screenshot 2023-01-19 at 11 27 26 AM](https://user-images.githubusercontent.com/100334861/213584543-6cfe2314-e59c-4550-8e81-643f47bd4fc2.png)

**Comment by tushar1-000** [comment]
I want to work on it but there is no read-me file for this project.
